### ISSUE TITLE
SimLife segment load failure

### ISSUE THREAD

**Issue by jschwartzenberg** [dosemu2/dosemu2#1529]
**Describe the bug**
At one point during the tutorial when following the suggestion to check the score, I got this:
![Screenshot_20210815_171201](https://user-images.githubusercontent.com/877112/129483293-9a04c89f-ced7-438e-ac34-b892f60423ec.png)

**To Reproduce**
Install the game & do the tutorial. When asked to 'Windows' -> 'Score' this showed up. I'll also check if that just reproduces it.

**Attach program/game binaries or provide an URL**
https://archive.org/details/SIMLIFE (need to double check this is the same version that I was testing with, but appears to be)

**Attach the log**
[boot.log](https://github.com/dosemu2/dosemu2/files/6988379/boot.log)

**A regression?**
No idea

**Additional info**
Only `$_cpu_vm_dpmi="native"`.

**Comment by stsp** [comment]
Try in 386

**Comment by jschwartzenberg** [comment]
Yeah I got way further in the tutorial now. Unfortunately in the second part of the tutorial I ran into this one:
![Screenshot_20210818_205146](https://user-images.githubusercontent.com/877112/129955477-20aa45e8-9350-4d25-b92f-7162ad6e257d.png)

**Comment by stsp** [comment]
And no problems with trident?

**Comment by jschwartzenberg** [comment]
I'll check, but I remember a similar crash in Civilization 2 with the Trident driver. Might win31 drivers generally be crappy?

**Comment by stsp** [comment]
Only another emulator can tell.

**Comment by jschwartzenberg** [comment]
Yeah I guess best thing to do would be to try these drivers also with PCem.

**Comment by stsp** [comment]
Is this still a problem?
IIRC the alike windows problems
were addressed in another ticket.

**Comment by stsp** [comment]
Ping.
Does vbesvga fixes this?
